content/blog/posts/cyberforce-2019-story.rst

```diff
@@ -0,0 +1,22 @@
+OSU Security Club Wins Regional Competition
+--------------------------------------------------
+:author: Abby Lawrence
+:date: 2020-02-04
+:img: 2019-Cyberforce-competition-1.jpg
```

Comment:
This photo needs to be resized so that it looks like it has the proper aspect ratio. You might look at the sizes we used in previous feature posts.